ames_qsar/out_fileread.py

```diff
@@ -0,0 +1,15 @@
+import os
+import glob
+
+def out_fileread(filepath):
+    os.chdir(filepath)

```

Comment:
this is called an unexpected side effect. The function returns a list of filenames matching the *.out glob, but it also changed the working directory of the application. Generally you do not want to do this because it can have unintended consequences. You'll want to either (a) make the function not require changing the dir, or you read the dir and change it back when done.